runtime/compiler/env/J9PersistentInfo.hpp

```diff
@@ -45,6 +45,42 @@ class TR_PersistentClassLoaderTable;
 class TR_J2IThunkTable;
 namespace J9 { class Options; }
 
+/**
+ * @brief The JitStates enum is used to represent the various states
+ *        the JIT can enter. The heuristics and behaviour of the JIT
+ *        is directly affected by the state it is in.
+ *
+ *        UNDEFINED_STATE: Uninitialized.
+ *
+ *             IDLE_STATE: The JVM isn't doing any work. This is when
+ *                         the application threads are doing no or
+ *                         minimal work.
+ *
+ *          STARTUP_STATE: The application is starting up; this includes
+ *                         the classloading phase. The heuristics in
+ *                         this state target ensuring the application
+ *                         starts up as quickly as possible.
```

Comment:
Doxygen won't pick these up unless the documentation is right above each enum declaration.

Reply:
Ah really; ok I'll look at what it should look like for enums.